README.md

```diff
@@ -1 +1,19 @@
 # art-gallery2
```

Comment:
Change the title ( heading ) and write using uppercase without a dash 
and after it adds the description.